packages/contracts/contracts/newsroom/Newsroom.sol

```diff
@@ -184,27 +191,44 @@ contract Newsroom is ACL {
     require(content.author == 0x0 || content.author == author);
     require(content.revisions.length > revisionId);
 
+    if (contentId == 0) {
```

Comment:
Is there anything special about contents[0]?

Reply:
@thec00n Yes, it's a newsroom charter, which is established during construction of a newsroom. That article is Owner-editable only and constitutes a "About page".

I shouldn't have added this change to this PR though. My bad